### DISCUSSION TITLE
Problem to use Crazyflie 1.0

### DISCUSSION THREAD

**Discussion by LePoulet76** [bitcraze/discussions#539] [Q&A]
Hey !
Since 2 week i'm trying to use crazyflie 1.0 but I can't connect them to my windows 11 (even with the old version of the software). When scanning, nothing happens. Can you help me ?

**Comment by ataffanel** [comment]
Hi,
This is not something any of us has tried recently but, in principle, I do not see anything that should not work controlling a Crazyflie 1 from a modern machine.

I have a couple of questions that can help us narrowing down what could be happening:
 1. What version of the lib/client are you using?
 2. Is the Crazyflie 1 starting properly and passing self test?
 3. Is the Crazyradio USB driver installed in Windows, do you see the RED led lighting up on the Crazyradio when you scan for Crazyflies?

**Reply by LePoulet76** [reply]
Hi ! 
Sorry for the time to answer. I tried to connect with the lastest version of the VM but even if it detects the drone, the application time out and so never connect to the drone. 
After more researches, i managed to connect and make my drone fly with a very old version of the application. So i think my drone is well connected and the crazyradio have the right driver.

Now, the main problem is to make them fly with code, But with the old version of the firmware, (and also the lib) the drone can't connect. I saw in the firmware that the python version used is the 2.7. Do you think i have to use python 2.7 to make the drone fly ?

**Reply by ataffanel** [reply]
This is odd, if the client connects, scripts should connect. Are you running the client from source, or a windows build? You should make sure you run the client from source using the same python interpreter.

It is indeed very likely that such old lib will require, or at least work, with Python 2.7.

**Reply by LePoulet76** [reply]
I don't really understand what do you mean by running the client from source or a windows build. For the python interpreter, i always use the 
same. I'll try to code with python 2.7. I'll come back to you.

**Reply by ataffanel** [reply]
There has been working "build" of the client in the past where the client was distributed as a Windows installer and you could run it from a .exe that bundles its own version of python.

So what I meant was that you should make sure you are not using one of those but instead that you are running the client with the same Python interpreter you are running your scripts.
